pkg/storage/gc_queue.go

```diff
@@ -143,10 +143,14 @@ func (r gcQueueScore) String() string {
 	if r.ExpMinGCByteAgeReduction < 0 {
 		r.ExpMinGCByteAgeReduction = 0
 	}
+	likelyLastGC := "never"
+	if r.LikelyLastGC != 0 {
```

Comment:
Negative values also produce a nonsensical message so maybe we should detect those and change the message to "unknown".